### ISSUE TITLE
[xenial] fix errors in functional tests

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4039]
## Description

The functional tests need updating to run under Xenial. Don't expect them all to pass until all other xenial app code updates have been completed. This ticket is scoped _just_ to getting them to either pass or fail under xenial - while not breaking them on trusty. 

Optional: Current Tor Browser is based on Firefox ESR 60, so I would encourage updating both the trusty and xenial jobs to Firefox ESR 60 for consistency with what we want to be testing against. 

## Steps to Reproduce

`make test-xenial`

## Expected Behavior

The tests either PASS or FAIL

## Actual Behavior

All the functional tests [ERROR](https://circleci.com/gh/freedomofpress/securedrop/20723), e.g.:

```
tests/functional/test_journalist.py::TestJournalist::test_journalist_interface_ui_with_modal ERROR 
```

**Comment by kushaldas** [comment]
We can keep all  of the functional tests fixed for now if we just use Firefox 46.0 like in Trusty. And then we can work on the Tor browser brunch on xenial to make it working here. Otherwise I have a PR (with ideas from the tbb branch) which fixes most of the issues, but, a few modal related ones are remaining. @redshiftzero thoughts?

**Comment by redshiftzero** [comment]
OK, we can go back to Firefox 46 to get them running. We _can_ work on the Tor Browser branch, though I expect that the path to getting passing functional tests on xenial is shorter in separate PR than getting the huge TB branch merged
